usagov/vote-gov-drupal
# VOTE-83: Add config files for NVRF Card block.

## Jira ticket

[VOTE-83](https://cm-jira.usa.gov/browse/VOTE-83)

## Description

Add new block type, NVRF Card.

## Deployment and testing

### Pre-deploy

None

### Post-deploy

1. For local environments, run `lando drush cim` to import the configurations for the new block type.

### QA/Test

1. Login as an admin
2. Goto the Content Block admin page and create a new NVRF Card block.
3. Confirm that the fields match the requirements within the following document https://docs.google.com/spreadsheets/d/1Z8j-a9KLrDlyG4kGqXmfwENlidtXTLYE/edit#gid=1963719072
4. Once created, confirm that the new NVRF Card can be translated.

## Checklist for the Developer




## Checklist for the Peer Reviewers
